libdrgn/register_state.h

```diff
@@ -130,6 +130,9 @@ struct drgn_register_state *drgn_register_state_create_impl(uint32_t regs_size,
 					DRGN_REGISTER_NUMBER(last_reg) + 1,	\
 					interrupted)
 
+struct drgn_register_state *drgn_register_state_dup(
+		const struct drgn_register_state *regs);
```

Comment:
Same style nit as above, and let's document this:

```suggestion
/**
 * Create a copy of a @ref drgn_register_state.
 *
 * @return New register state on success, @c NULL on failure to allocate memory.
 */
struct drgn_register_state *
drgn_register_state_dup(const struct drgn_register_state *regs);
```